src/applications/fry-dea/sass/fry-dea.scss

```diff
@@ -129,7 +129,7 @@ $error-left-padding-width: 1.5rem;
 // END - Benefit Selection Page
 
 // Confirmation Page: Under Review
-.meb-hr#custom-fry-hr {
+#custom-fry-hr {
```

Comment:
In general, we should always use classes and only use IDs with rare exception. Two reasons:
- using an ID prevents us from using this element/applying this style twice on the same page;
- if we need to override any part of this styling, we can't do it with a class.

Reply:
Reverted to class based styling.